contrib_bots/bots/translate/readme.md

```diff
@@ -0,0 +1,17 @@
+To setup the bot, you will first need to move translate_bot.config into the user home directory and add an API key.
```

Comment:
You might want to look at the other bots in `contrib_bots` for a template of how to write the `readme.md` file for bots. You should summarize how to use the commands of your bot.